google/cloud/storage/internal/grpc_object_request_parser.cc

```diff
@@ -260,6 +260,44 @@ ResumableUploadResponse GrpcObjectRequestParser::FromProto(
   return response;
 }
 
+google::storage::v2::ListObjectsRequest GrpcObjectRequestParser::ToProto(
+    ListObjectsRequest const& request) {
+  google::storage::v2::ListObjectsRequest result;
+  result.set_parent("projects/_/buckets/" + request.bucket_name());
+  auto const page_size = request.GetOption<MaxResults>().value_or(0);
+  // Clamp out of range values, the service will clamp to its own range ([0,
```

Comment:
nit: s/, the/. The/

I think the clamping of `page_size` might be overly cautious.... but I trust your judgment. If you want to be super-duper cautious you could add tests for the different cases. (I am not recommending this).

Reply:
Ack.